lib/soapy_cake/admin.rb

```diff
@@ -109,11 +109,16 @@ def update_campaign(opts = {})
     def add_blacklist(opts)
       require_params(opts, %i(blacklist_date))
 
+      blacklist_date = opts[:blacklist_date]
```

Comment:
We can add a 'to_date' to be sure. The format should be fine, used to be like that all the time.